packages/react-components/babel-preset-storybook-full-source/src/modifyImports.ts

```diff
@@ -49,7 +49,24 @@ export function modifyImportsPlugin(babel: typeof Babel, options: BabelPluginOpt
       // eslint-disable-next-line @typescript-eslint/naming-convention
       ImportDeclaration(path, pluginState) {
         let importSource = path.node.source;
-        if (importSource.value.startsWith('.')) {
```

Comment:
this was naive logic, as any import paths was considered to be import from package API which might not be the case.